frozendict/frozendict.pyi

```diff
@@ -1,6 +1,57 @@
-try:
-    from ._frozendict import *
-except ImportError:
-    from .core import *
+import collections
+from _typeshed import Self
+from collections.abc import Iterable, Iterator, Mapping
+from typing import Any, Generic, NoReturn, Tuple, TypeVar, Union, overload
 
-FrozenOrderedDict = frozendict
+_KT = TypeVar("_KT")
+_VT = TypeVar("_VT")
+
+def frozendict_or(
+    self: Mapping[_KT, _VT], other: Mapping[_KT, _VT]
+) -> "frozendict[_KT, _VT]": ...
+
+class frozendict(Mapping[_KT, _VT], Generic[_KT, _VT]):
+    # Fake __init__ to describe what __new__ does:
+    dict_cls: type[dict[Any, Any]] = ...
+    @overload
+    def __init__(self, **kwargs: _VT) -> None: ...
+    @overload
+    def __init__(self, mapping: Mapping[_KT, _VT]) -> None: ...
+    @overload
+    def __init__(self, iterable: Iterable[tuple[_KT, _VT]]) -> None: ...
+
+    # Magic Methods:
+    def __getitem__(self, __key: _KT) -> _VT: ...
```

Comment:
Excuse me for annoying questions, but why double underscore in arg, and only here?